app/src/main/res/values/strings.xml

```diff
@@ -124,8 +122,8 @@
     <string name="attack">Attack</string>
     <string name="defense">Defense</string>
     <string name="stamina_hp">HP</string>
-    <string name="missing_inputs">Missing data. Please fill CP and HP inputs.</string>
-    <string name="ivtext_no_possibilities">There are no IV possible combinations for this input. Please double check the information.</string>
+    <string name="missing_inputs">CP and HP are required.</string>
+    <string name="ivtext_no_possibilities">There are no possible combinations. Make sure the values you have entered are correct.</string>
```

Comment:
Thanks for your comments, I'll keep them in mind.

> "Make sure your inputs are correct" doesn't sound as professional.

I think I know what you mean, but I've been trained out of that style in my writing courses, following the tradition set for instance by George Orwell.